allennlp/training/metrics/average.py

```diff
@@ -25,8 +26,10 @@ def __call__(self, value):
         value : `float`
             The value to average.
         """
-        self._count += dist_reduce(1, ReduceOp.SUM)
-        self._total_value += dist_reduce(float(list(self.detach_tensors(value))[0]), ReduceOp.SUM)
+        self._count += dist_reduce(1, dist.ReduceOp.SUM)
+        self._total_value += dist_reduce(
+            float(list(self.detach_tensors(value))[0]), dist.ReduceOp.SUM
+        )
```

Comment:
If `distributed` is not installed, this here still fails though, right?

Reply:
Ah yeup, good catch.